src/Lumi/Components/DropdownButton.purs

```diff
@@ -232,6 +235,38 @@ dropdownMenuDefaults =
   , items: []
   }
 
+type DropdownIconProps =
+ { icon :: JSX
+ , content :: JSX
+ , onOpen :: Effect Unit
+ , alignment :: Nullable String
+ }
+
+dropdownIconDefaults :: DropdownIconProps
+dropdownIconDefaults =
+  { icon: Text.text
+      { children: [ R.text "···" ]
+      , className: Nullable.null
+      , color: Nullable.notNull colorNames.black1
+      , style: R.css { fontSize: "18px", fontWeight: "bold" }
+      , tag: "lumi-body"
+      , testId: Nullable.null
+      }
+  , content: mempty
+  , onOpen: pure unit
+  , alignment: toNullable Nothing
+  }
+
+dropdownIcon :: DropdownIconProps -> JSX
+dropdownIcon props =
+  dropdownButton
+    { label: (unsafeCoerce :: JSX -> String) props.icon
```

Comment:
This unnerves me a little, what happens if `props.icon` isn't a `Text`? Do we really just want the `label` prop of the dropdownButton props to be `JSX`?

Reply:
Yes, but then Button has to change for the same reason and I considered doing that but it'd be a large change. I'd like to do it as a more comprehensive "remove string props" PR once we aren't using them from JS anymore.

Reply:
Ah okay, so for now this will work the same way as if `ButtonProps`' `label` prop were already `JSX`?

Reply:
Ah right yes, I've just checked and I see what's going on now. :+1:

Reply:
I added a comment